### ISSUE TITLE
Ignite - No timeout on cache:put

### ISSUE THREAD

**Issue by GlennRicaud** [enonic/xp#6401]
Current situation:
Block one node of the cluster (debug on IgniteSessionDataStore for example)

The other node gets blocked on igniteCache.put for 10 minutes (The failing node seems to have been removed from the cluster).
Use ignite.putAsync with a timeout.

If linked investigate these two other issues, otherwise, create separate issue:
- On startup ther eis a mention of no SessionScavenger set => Scavenging every 600000ms (10mins)
- When the failing node was restarted it did not manage to join the Ignite cluster again
